src/Microsoft.Extensions.Logging.Console/ConsoleLogger.cs

```diff
@@ -137,12 +150,12 @@ public virtual void WriteMessage(LogLevel logLevel, string logName, int eventId,
             if (exception != null)
             {
                 // exception message
-                exceptionText = exception.ToString();
-                printLog = true;
+                sb.AppendLine(exception.ToString());
             }
 
-            if (printLog)
+            if (sb.Length > 0)
             {
+                var msg = sb.ToString();
```

Comment:
give this a better name. `message` is already used, and this isn't semantically the same as `message`. This is a pretty good example of why we don't abbreviate :laughing: